drivers/gl/vertex_stream.go

```diff
@@ -5,36 +5,41 @@
 package gl
 
 import (
+	"encoding/binary"
 	"fmt"
 	"reflect"
 
-	"github.com/go-gl/gl/v2.1/gl"
+	"github.com/goxjs/gl"
+	"golang.org/x/mobile/f32"
```

Comment:
Yes, we can if desired. It is only used in this one file for its [`Bytes`](https://godoc.org/golang.org/x/mobile/f32#Bytes) helper func to convert `[]float32` into a corresponding `[]byte` input to `gl.BufferData`.

What are your suggestions on how to do that? Would you prefer I copied the `golang.org/x/mobile/f32` package as a subfolder in `github.com/goxjs/gl`? Or would you like to copy the `Bytes` func into `gxui` itself as an unexported helper?